modules/exploits/multi/http/apache_jetspeed_upload.rb

```diff
@@ -0,0 +1,195 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+
+  Rank = ManualRanking
```

Comment:
Only reason, yeah. It basically destroys the web interface, but it gets you a shell. I'm sure there's a way to fix the ZIP so that doesn't happen.